client/src/reducers/teamReducer.js

```diff
@@ -42,7 +43,11 @@ const teamReducer = (state = INITIAL_TEAM_STATE, action) => {
       return { ...state, twitterHandle: action.payload };
     case UNLINK_TEAM_TWITTER:
       return { ...state, twitterHandle: action.payload };
+    case GET_GH_ACCESS_TOKEN:
+      return { ...state, retrievedAccessToken: true };
     case ADD_TEAM:
+      return { ...action.payload };
+    case UPDATE_TEAM:
```

Comment:
There's some listing issues here, that might be nice to clean before merging into `main` eventually